src/js/tabs/firmware_flasher.js

```diff
@@ -60,12 +60,18 @@ firmware_flasher.initialize = function (callback) {
             } else {
                 $('div.release_info #manufacturerInfo').hide();
             }
+
             $('div.release_info .target').text(summary.target);
             $('div.release_info .name').text(summary.release).prop('href', summary.releaseUrl);
             $('div.release_info .date').text(summary.date);
             $('div.release_info #targetMCU').text(summary.mcu);
             $('div.release_info .configFilename').text(self.isConfigLocal ? self.configFilename : "[default]");
 
+            // Wiki link to "BOARD---" + target
+            let targetWiki = $('#targetWikiInfoUrl');
+            targetWiki.html(`[Wiki: Board---${summary.target}]`);
```

Comment:
If we are going to set this up anew then we could start with just single `-` ??? @blckmn

Reply:
No just use the property `summary.wiki` if it is defined then display the wiki link, else hide it.

Reply:
so something like this: 
```
if (summary.wiki) {
     targetWiki.html('[Wiki]');
     targetWiki.attr("href", summary.wiki);
} else {
    targetWiki.html('');
} 
```

Reply:
No issue to change number of dash, but maybe just ex. [Wiki: Board FRSKYF4]", this to promote the benefits of documenation at the Wiki. This may also give the code snip:
     targetWiki.html('[Wiki]');
     if (summary.wiki) {
          targetWiki.attr("href", summary.wiki);
     } else {
         targetWiki.html("href", 'https://github.com/betaflight/betaflight/wiki');  // can be an entry in message.jason
     }